addons/web/static/src/webclient/commands/command_palette.js

```diff
@@ -237,6 +237,7 @@ export class CommandPalette extends Component {
 
     async executeSelectedCommand() {
         await this.inputPromise;
+        await this.keepLast;
```

Comment:
These two lines seem confusing and could use a bit of explanations

Reply:
Mostly, this line is useless, this isn't how keepLast works (keepLast isn't a promise, it's a class with an "add" function, that returns a promise